### PR TITLE
Add a Los Angeles Image 

### PR DESCRIPTION
**Pull request by hunterowens**
This PR adds the Los Angeles Metro as an example project for other users to add and contribute to. 

We greatly appreciated the work from the Pelias team here at @cityoflosangeles. Wanted to contribute this back to anybody who might need LA area data.

### CODE REVIEW

**Comment by orangejulius** [comment]
Hey this is awesome! Really cool to see you added lots of transit feesd as well. @fpurcell and the folks at Trimet who contributed that code will be glad to hear it's already taking off.

File: projects/los-angeles-metro/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+COMPOSE_PROJECT_NAME=pelias
+DATA_DIR=/mnt/data/pelias-data/los-angeles-metro
```
Comment (missinglink) [inline projects/los-angeles-metro/.env]: can we please default this to `/tmp/pelias/los-angeles-metro`?

File: projects/los-angeles-metro/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
+# Portland Metro Area
```
Comment (missinglink) [inline projects/los-angeles-metro/README.md]: There are a few references to Portland in the readme, also one more in the next sentence.

File: projects/los-angeles-metro/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+pelias prepare all
+pelias import all
+pelias compose up
+```
+
+# Make an Example Query
+
+You can now make queries against your new Pelias build:
+
+http://localhost:4000/v1/search?text=pdx
```
Comment (missinglink) [inline projects/los-angeles-metro/README.md]: LAX?

**Comment by missinglink** [comment]
@hunterowens this is great, thanks very much for the contribution!

**Pull request merged**